### PR TITLE
Updated changelog v0.16

### PR DESCRIPTION
**Pull request by pamil**
Due to #3110.

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
@pamil Requires rebasing.

**Comment by pamil** [comment]
@pjedrzejewski done :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+## v0.16 (2015-09-xx)
 
 This chapter references the relevant changes done in 0.16 version.
 
 To get the diff between two versions, go to https://github.com/Sylius/Sylius/compare/v0.15.0...master
 
+ * feature #3100 [BC BREAK] Bumped minimal versions, major changes: PHP >=5.5.9, Symfony ^2.7
 
-## v0.15 (2015-09-08)
+## v0.15 (2015-07-xx)
```
Comment (michalmarcinkowski) [inline CHANGELOG.md]: This should be reverted.

**Comment by pjedrzejewski** [comment]
:+1:

**Pull request merged**